src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -294,7 +294,7 @@ g.test('sampleCount,valid_sampleCount_with_other_parameter_varies')
       .unless(({ sampleCount, usage, format, mipLevelCount, dimension }) => {
         const info = kTextureFormatInfo[format];
         return (
-          ((usage & GPUConst.TextureUsage.RENDER_ATTACHMENT) !== 0 && !info.renderable) ||
+          ((usage & GPUConst.TextureUsage.RENDER_ATTACHMENT) !== 0 && (!info.renderable || dimension !== '2d')) ||
```

Comment:
FYI you can fix the warning that was added by Github actions here with `npm run fix`.

Reply:
Also I think we should update the `texture_usage` test in the same file (around line 700) so that it checks that validation rule.